build.cake

```diff
@@ -148,16 +172,21 @@ Task("Test")
             case "testnet452": settings.Framework = "net452"; break;
             case "testnetstandard15": settings.Framework = "netcoreapp1.1"; break;
             case "testnetstandard20": settings.Framework = "netcoreapp2.1"; break;
+            case "testnetcoreapp3.1": settings.Framework = "netcoreapp3.1"; break;
+        }
+        if (IsSuiteRunnable(settings.Framework))
```

Comment:
I don't understand this check.

You are skipping the ENTIRE test suite just because an OCSP environment variable exists?

Shouldn't we be running the test suite and just skipping as necessary the individual tests that are OCSP related?

Reply:
From my understanding there are not OCSP specific tests. It just changes the connection string, adds a few environment variables, changes the running cluster, and then runs the entire test suite. So as you can see in the current failures for OCSP all tests are failing that actually hit the database with auth errors.